analysis/analysis/ic50.py

```diff
@@ -0,0 +1,65 @@
+import json
```

Comment:
You can run `isort ic50.py` in terminal so that the imports are structured in a nice way.